build.cmd

```diff
@@ -1,5 +1,5 @@
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT License.
+REM Copyright (c) Microsoft Corporation.
+REM Licensed under the MIT License.
```

Comment:
(nit) As I remember the `REM` commands are typically prefixed with `@` (`@REM`) for them to not be displayed in the console during the script run.

Reply:
Done!